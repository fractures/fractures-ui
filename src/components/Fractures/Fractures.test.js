// import Fractures from "./Fractures.js"

xdescribe("Fractures", () => {
	const tree = renderer.create(<Fractures />).toJSON()

	it("Renders correctly", () => expect(tree).toMatchSnapshot())
})

xdescribe("Fractures with theme", () => {
	const tree = renderer.create(<Fractures theme="blue" />).toJSON()

	it("Renders correctly", () => expect(tree).toMatchSnapshot())
})
