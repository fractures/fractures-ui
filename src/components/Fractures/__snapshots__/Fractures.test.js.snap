// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fractures Renders correctly 1`] = `null`;

exports[`Fractures with theme Renders correctly 1`] = `
<style
  dangerouslySetInnerHTML={
    Object {
      "__html": "
		:root {
			--fr-ground: #fff;
			--fr-focus: rgba(52, 130, 218, 0.4);
			--fr-100: #e8f2fd;
			--fr-300: #76a7e7;
			--fr-500: #3482da;
			--fr-700: #2463a5;
			--fr-900: #102a45;

			--fr-color-red: #f94b56;
			--fr-color-green: #82c971;
			--fr-color-yellow: #ffd058;

			--fr-font-mono: \\"Roboto Mono\\", monospace;
		}
	",
    }
  }
/>
`;
